sonar-project.properties

```diff
@@ -19,6 +19,7 @@ sonar.sources= \
 sonar.exclusions= \
     **/test/**, \
     **/tests/**, \
+    **/e2e-tests/**, \
```

Comment:
Why are we excluding tests from sonarqube?

Reply:
because the test code itself should not be included in the coverage metrics.

Reply:
why not?

Reply:
It's standard practice as described in the sonarqube documentation [here](https://docs.sonarqube.org/9.8/project-administration/narrowing-the-focus/). Tests should be excluded from source scope and included under test scope instead